source/_integrations/vacuum.mqtt.markdown

```diff
@@ -140,6 +140,10 @@ name:
   required: false
   type: string
   default: MQTT Vacuum
+object_id:
```

Comment:
Please update both schemas (legacy, state) with the new parameter.

Reply:
Done (add in state)